django/contrib/gis/gdal/raster/band.py

```diff
@@ -19,6 +19,14 @@ def __init__(self, source, index):
         self.source = source
         self._ptr = capi.get_ds_raster_band(source._ptr, index)
 
+    def _flush(self):
+        """
+        Calls the flush method on the Band's parent raster and
```

Comment:
For new docstrings, we are using PEP 0257 verb style ("Call.... and clear...")
Below, "Return..."